sdk/communication/azure-communication-chat/azure/communication/chat/aio/_chat_thread_client_async.py

```diff
@@ -382,6 +385,7 @@ async def update_message(
             self,
             message_id: str,
             content: str = None,
+            metadata: Dict[str, str] = None,
```

Comment:
As I mentioned above - for the async method signature we should use `*` to mark this as keyword-only:
```
async def update_message(
            self,
            message_id: str,
            content: str = None,
            *,
            metadata: Dict[str, str] = None,
            **kwargs
    ) -> None:
```